recipe/meta.yaml

```diff
@@ -55,6 +62,7 @@ outputs:
       run:
         - {{ pin_subpackage("rust-std-" ~ rust_arch, exact=True) }}
         - gcc_impl_{{ target_platform }}  # [linux]
+        - __osx >={{ MACOSX_DEPLOYMENT_TARGET|default("10.9") }}  # [osx and x86_64]
```

Comment:
This should be moved to `run_exports`

Reply:
This doesn't make sense in `run` because the compiler itself can be run in 10.9, only the compiled binaries cannot be run which means we need `run_exports`.

Reply:
Makes sense.
Are we comfortable in adding virtual packages to `run` generally? Reading https://github.com/conda/conda/blob/4.9.2/CHANGELOG.md#bug-fixes-2 it's still rather recent that bugs have been fixed regarding them.
Do we want `run_exports: { strong: ... }` or be rather conservative (in expecting that users may run older non-fixed `conda` versions) and use `run_exports: { strong_constrains: ... }`?

Reply:
> Do we want run_exports: { strong: ... } or be rather conservative (in expecting that users may run older non-fixed conda versions) and use run_exports: { strong_constrains: ... }?

Sure. I was gonna say no such thing exists, but it does. :smile: https://github.com/conda/conda-build/pull/4125